corehq/apps/app_manager/templates/app_manager/v2/partials/case_list.html

```diff
@@ -62,7 +63,7 @@ <h4 class="panel-title panel-title-nolink">
             </div>
         </div>
         {% endif %}
-        <div data-bind="visible: useCaseTiles() == 'yes' || COMMCAREHQ.toggleEnabled('CASE_LIST_CUSTOM_XML')">
+        <div data-bind="visible: (useCaseTiles() == 'yes' && COMMCAREHQ.toggleEnabled('CASE_LIST_TILE')) || COMMCAREHQ.toggleEnabled('CASE_LIST_CUSTOM_XML')">
```

Comment:
Why is this change necessary? Changing this doesn't seem like part of the feature you're adding and I think the original logic is fine.